### PR TITLE
bug fixed  label Problems with using stroke and shadow

### PR DESCRIPTION
**Pull request by ankye**
https://github.com/cocos/cocos-engine/issues/16666

Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/2d/assembler/label/text-processing.ts [hunk 1]
```diff
@@ -470,15 +491,26 @@ export class TextProcessing {
         const tempPos = new Vec2(outputLayoutData.startPosition.x, outputLayoutData.startPosition.y);
         const drawTextPosX = tempPos.x;
         let drawTextPosY = 0;
+        const tempFillStyle = this._context.fillStyle;
```
Comment (qiuguohua) [inline cocos/2d/assembler/label/text-processing.ts]: This variable is not used?

File: cocos/2d/assembler/label/text-processing.ts [hunk 2]
```diff
@@ -573,6 +607,10 @@ export class TextProcessing {
     }
 
     private _setupOutline (style: TextStyle): void {
+        this._context!.shadowBlur = 0;
```
Comment (qiuguohua) [inline cocos/2d/assembler/label/text-processing.ts:610]: Is it possible to add a note？

File: cocos/2d/assembler/label/text-processing.ts [hunk 3]
```diff
@@ -573,6 +607,10 @@ export class TextProcessing {
...
+        this._context!.shadowOffsetX = 0;
+        this._context!.shadowOffsetY = 0;
+        this._context!.miterLimit = 2;
```
Comment (qiuguohua) [inline cocos/2d/assembler/label/text-processing.ts]: Why do we need to set miterLimit to 2?

File: cocos/2d/assembler/label/text-processing.ts (hunk 1 above)
Comment (ankye) [reply cocos/2d/assembler/label/text-processing.ts]: Test code forgot to delete clean

File: cocos/2d/assembler/label/text-processing.ts (hunk 3 above)
Comment (ankye) [reply cocos/2d/assembler/label/text-processing.ts]: Hoping for a clearer stroke, no effect，i will remove

**Review by qiuguohua — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**